### ISSUE TITLE
Gradle dependency issue in v3.0.2 with Guava and Findbugs

### ISSUE THREAD

**Issue by psimonazzi** [dropwizard/dropwizard#7592]
We encountered an issue in the dependencies of v3.0.2 when building a project with gradle. 
With v3.0.1 everything was fine.
The issue is related to the Findbugs dependency of Guava (which is excluded in the Drowizard pom):

```
Could not find com.google.code.findbugs:jsr305:.
     Required by:
         project : > io.dropwizard:dropwizard-core:3.0.2 > io.dropwizard:dropwizard-jackson:3.0.2 > com.google.guava:guava:32.1.2-jre
```

Attached you can find a minimal project to reproduce the issue.
It contains just a main class and the only dependency is dropwizard-core v3.0.2. 

Build with `gradlew build` to check the error. 
If you change the dropwizard-core version to v3.0.1 the project builds correctly.
Including only the guava dependency without dropwizard does not cause the issue, which is why I'm reporting here.

(Edit: seems to be acknowledged by Guava: https://github.com/google/guava/issues/6654 but a fix is not released yet?)

[dropwizard-test.zip](https://github.com/dropwizard/dropwizard/files/12763447/dropwizard-test.zip)

**Comment by psimonazzi** [comment]
Fixed in v3.0.3 with Guava 32.1.3

**Comment by joschi** [comment]
@psimonazzi Thanks for your update!
